android/src/main/java/com/reactnativetwiliovideo/TwilioVideoModule.kt

```diff
@@ -96,6 +97,23 @@ class TwilioVideoModule(reactContext: ReactApplicationContext) : ReactContextBas
     }
   }
 
+  @ReactMethod
+  fun listCameras(promise: Promise) {
+    val camera1Enumerator = Camera1Enumerator()
+    val cameraResults = Arguments.createArray()
+    camera1Enumerator.deviceNames.forEach {
```

Comment:
I don't love how this is a procedural monolith but

no abstraction presented itself to me, since we're starting from a pretty poor OO abstraction